### PR TITLE
oh-my-posh 7.18.0 (new formula)

### PR DESCRIPTION
**Pull request by paperchalice**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/oh-my-posh.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X main.Version=#{version}
+    ]
+    cd "src" do
+      system "go", "build", *std_go_args(ldflags: ldflags)
+    end
+    prefix.install "themes"
```
Comment (carlocab) [inline Formula/oh-my-posh.rb:19]: Should this maybe be in `pkgshare`?

File: Formula/oh-my-posh.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X main.Version=#{version}
+    ]
+    cd "src" do
+      system "go", "build", *std_go_args(ldflags: ldflags)
+    end
```
Comment (carlocab) [inline Formula/oh-my-posh.rb:18]:
```suggestion
    system "go", "build", *std_go_args(ldflags: ldflags), "./src"
```
This should be equivalent, no?

Comment (paperchalice) [reply Formula/oh-my-posh.rb:18]:
```
go build -trimpath -o=/usr/local/Cellar/oh-my-posh/7.18.0/bin/oh-my-posh -ldflags=-s -w -X main.Version=7.18.0 ./src
  go: go.mod file not found in current directory or any parent directory; see 'go help modules'
```

**Review by carlocab — APPROVED** [review]
Thanks!

**Pull request closed without merging**